### ISSUE TITLE
Various issues with controls when Quick Loot RE activated in load order

### ISSUE THREAD

**Issue by dunc001** [Ryan-rsm-McKenzie/QuickLootRE#78]
Can no longer use keyboard controls with Gamepad enabled (using Gamepad++ controlmap which has dual input enabled).  With Quick Loot none of the keyboard inputs are recognised so can't open console, journal, etc

Player controls are completely frozen out during animations (mining, bathing, etc) so can't move camera when these are playing and can't cancel out of an animation by doing something else.  Also can't place Campfire tents/campfires as the controls stop responding as soon as the glowing tent appears with the 'Place Here' menu.

Can't move camera when in dialogue (same issue as above I guess).

Have tested on a vanilla load order and confirmed these issues are only present with Quick Loot enabled.  These issues are present all the time, not just when the Quick Loot menu is showing.

**Comment by EASK8ER52** [comment]
Yes I am having the same issue. Took me forever to figure this out.

**Comment by SpikerzZ406** [comment]
Same here all though I never tested mining yet but the other ones yes

**Comment by Ryan-rsm-McKenzie** [comment]
Quickloot disables input using control groups. The custom controlmap that ships with gamepad++ does not have all vanilla control groups set.

**Comment by dunc001** [comment]
Why is it affecting anything when the menu isn't displayed?  I can understand (well I can't really) why you may want to lock out controls while the menu is open, but surely it shouldn't be having an effect all the time?  It's years since that controlmap was compiled, I was under the impression that the group set flags had to be removed to allow both controller and keyboard controls to remain active at the same time (iirc that nugget was taken from another controlmap mod at the time).  Given you probably understand those flags better than me what needs to be done to the G++ controlmap to retain it's dual input functionality and remove any 'incompatibility' with Quick Loot?

**Comment by EASK8ER52** [comment]
I'm not sure about gamepad++ I don't use it. I was using extended hotkey system with my controller and it worked fine, then I switched back to Hotkeys with Controller and it would not work. Had to disable and enable all my mods until I found the culprit. Hotkeys with Controller only makes changes to the controlmap.txt located at \interface\controls\pc. Hopefully this helps you figure out what is happening. In the mean time I'll switch back to regular quick loot, Hotkeys with Controller works fine with it.
